### PR TITLE
Us22 eonet #39

### PR DESCRIPTION
**Pull request by ClemDSC**
PR asked prevously on main and NOT dev, sorry!

### CODE REVIEW

**Comment by gnos28** [comment]
ok

**Comment by gnos28** [comment]
ok

**Review by gnos28 — APPROVED** [review]
ok

File: package.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
     "husky": "^7.0.0"
   },
   "dependencies": {
-    "react-lazy-load-image-component": "^1.5.4"
+    "globe.gl": "^2.26.1",
+    "moment": "^2.29.3",
+    "react-lazy-load-image-component": "^1.5.4",
+    "save": "^2.5.0"
```
Comment (lchastanet) [inline package.json:25]: Remove this please

**Review by lchastanet — CHANGES_REQUESTED** [review]

**Pull request merged**